solution.cpp

```diff
@@ -1,23 +1,189 @@
-/* ----------------------------------------------------------------------------
- * Copyright &copy; 2015 TODO(your name) <TODO(your email)>
- * Released under the [MIT License] (http://opensource.org/licenses/MIT)
- * ------------------------------------------------------------------------- */
+/*-----------------------------------------------------------------
+* Copyright &copy; Brandon Min <brandon.min@csu.fullerton.edu>
+* Released under the [MIT License] (http://opensource.org/licenses/MIT)
+*---------------------------------------------------------------------*/
 
-/**
- * TODO: program description
- */
+//Assignment 1: Roman Numerals
 
-// TODO: `#include`s and function prototypes
+#include <iostream>
+#include <string>
+#include <stdlib.h>
 
-// ----------------------------------------------------------------------------
+using std::cout;
+using std::cin;
+using std::endl;
+using std::string;
 
-int main() {
-    // TODO
 
-    return 0;  // success
-}
+int main(){
+
+
+	string romannum;
+	int integer;
+	int x;
+	char restart;
+
+	//Asks user for if he/she would like to use the program and then for the integer.
+	
+
+	cout << "Would you like to convert an integer into roman numeral (Y/N)? " << endl;
+
+
+	while (cin >> restart){
+
+
+		if (restart == 'y' || restart == 'Y')
+		{
+
+			cout << endl;
+			cout << "Please enter an integer: ";
+			cin >> integer;
+
+			cout << "---> ";
+
+			if ((integer >= 4000) || (integer <= 0)){
```

Comment:
most of what's below should have been in a separate function -- it would make the code easier to read -- but no sense in changing it now :-)